packages/client/hmi-client/src/components/model/petrinet/tera-initial-entry.vue

```diff
@@ -1,32 +1,40 @@
 <template>
-	<header>
-		<strong>{{ initialId }}</strong>
-		<span v-if="name" class="ml-1">{{ '| ' + name }}</span>
-		<span v-if="unit" class="ml-2">({{ unit }})</span>
-		<span v-if="description" class="ml-4">{{ description }}</span>
-	</header>
-	<main>
-		<span class="expression">
+	<div>
+		<header>
+			<div>
+				<strong>{{ initialId }}</strong>
+				<span v-if="name" class="ml-1">{{ '| ' + name }}</span>
+				<span v-if="unit" class="ml-2">({{ unit }})</span>
+				<template v-if="unit">
```

Comment:
Is displaying unit information twice here intentional?

Reply:
Cole removed it in one of the later commit, so it probably didn't show up for you.

Reply:
sorry forgot to comment that I removed it...